common/persistence/serialization/blob.go

```diff
@@ -125,6 +125,18 @@ func QueueStateFromBlob(blob []byte, encoding string) (*persistencespb.QueueStat
 	return result, proto3Decode(blob, encoding, result)
 }
 
+func WorkflowExecutionRequestIDsToBlob(requestIDs *persistencespb.WorkflowExecutionRequestIDs) (*commonpb.DataBlob, error) {
+	return proto3Encode(requestIDs)
+}
+
+func WorkflowExecutionRequestIDsFromBlob(blob []byte, encoding string) (*persistencespb.WorkflowExecutionRequestIDs, error) {
+	if blob == nil {
+		return nil, nil
+	}
+	result := &persistencespb.WorkflowExecutionRequestIDs{}
+	return result, proto3Decode(blob, encoding, result)
+}
```

Comment:
This is code duplication for now.
We need to refactor the code to pass the serializer object to the DB plugins to be able to serialize/deserialize fields from DB.

Reply:
Will you do the refactor? Not blocking this PR, but ideally we'd at least track this.